sdk/python/v1beta1/setup.py

```diff
@@ -38,8 +38,10 @@
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
```

Comment:
I think we can also remove these Python <= 3 versions, since they are not longer maintained.

Reply:
Done.